.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules/
 dist/
 build/
 .firebase/
+apiKeys.json
```

Comment:
this should be an absolute URL
`src/javascripts/helpers/apiKeys.json`

Reply:
done